re-former pull request

sergiocortessat opened the pull request:
The request corresponds to the re-former activity

Pull request for the Forms project. 

NOTE: THIS PROJECT HAS BEEN BUILT SOLO as my coding partner has requested a month's leave.

I have added the required parts of the project. Such as the ability to show, edit, update, and new. I. I have added a simple bootstrap to make the page look better and a simple navbar header alongside simple tables and form controls. I have added an index as well, although this was not required by the project

This exercise complies with the requirements of creating, editing and reviwing

Commit:
Add Users model and controller

Commit:
Add classic form tag

Commit:
Add HTML form, params

Commit:
Add user_params to create

Commit:
Add form_tag

Commit:
Add form_with

Commit:
Add edit, show

Commit:
Add index and tables

Commit:
Add header

Commit:
Add readme, linters

Commit:
Add modification routes

Commit:
Add logic fixtures

Commit:
Add linters fix

Commit:
Add screenshot

descholar-ceo reviewed (changes requested):
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @sergiocortessat  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implement all of the minimal requirements of this project, keep it up, and your project is really almost done.
:+1: All of the linters are properly set and they show no errors,
:+1: Professional repository
:+1: Almost everything is working as it should
#### :warning: However we need to make few changes to mark this project approved. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 

- [x] Your repository is professional when all of the information on the readme file are accurate, currently you have this `Project Link: Ruby-capstone` could you change the `Ruby-capstone` to this project's name?

- [x] In user controller on create action you should redirect_to new_user_path if user was saved ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) from Odin)

- [x] In user controller on the update action you should redirect_to edit_user_path(<USER_ID>) if user was saved

- [x] On the `new` view of this file `app/views/users/new.html.erb` you should have used users_path helper for action instead of using `/users` on the commented out html raw form.

- [x] On the form_tag, you should have `form_tag(users_path)`
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

Commit:
Add TSE requirements

sergiocortessat commented:
Pull request 2
Note: This project has been submitted SOLO as my partner requested months leave

I have added to the project the required changes. As well I have added simple extra functionality to display error messages if any of the edit or new forms are subimmited with blanc spaces.

Commit:
Add licence

Commit:
Add licence readme

Commit:
Add readme mistake

sergiocortessat committed:
Create LICENSE

sergiocortessat commented:
Changes added to the readme LICENSE as requested
